lib/rspec/buildkite/analytics/ci.rb

```diff
@@ -14,13 +14,15 @@ def self.env
         "number" => ENV["BUILDKITE_BUILD_NUMBER"],
         "job_id" => ENV["BUILDKITE_JOB_ID"],
         "message" => ENV["BUILDKITE_MESSAGE"],
-        "debug" => ENV["BUILDKITE_ANALYTICS_DEBUG_ENABLED"]
+        "debug" => ENV["BUILDKITE_ANALYTICS_DEBUG_ENABLED"],
+        "version" => RSpec::Buildkite::Analytics::VERSION
```

Comment:
As new collectors show up we could get conflicting versions. I'm wondering if this should be
```suggestion
        "collector" => "RSpec - #{RSpec::Buildkite::Analytics::VERSION}"
```
or something?

Reply:
or maybe a new key of "collector" that has "RSpec in it.